webapp/src/app/ui/app-issue-card/app-issue-card.component.html

```diff
@@ -0,0 +1,31 @@
+<div class="border border-gray-300 rounded-lg p-4 w-72">
+  <div class="flex justify-between items-center mb-2 text-xs text-gray-500">
+    <span class="font-bold flex justify-center items-center space-x-1">
+      <ng-icon [svg]="octGitPullRequest" size="16" class="mr-1"></ng-icon> {{ pullRequest().repository.name }} #{{ pullRequest().number }} on {{ pullRequest().createdAt }}
```

Comment:
You have to handle the `null`-case for the repository here to avoid a compiler error.

I'd be fine with simple optional chaining:
`{{ pullRequest().repository?.name }}`